src/Navigation/SidePanel/__snapshots__/SidePanel.spec.tsx.snap

```diff
@@ -155,24 +159,12 @@ exports[`SidePanel default 1`] = `
 <div
   className="c0"
   onAnimationEnd={[Function]}
```

Comment:
like the idea of "end" I think is my favorite way to address animation. However, i do not like animations, found it confusing, click - get my favorite so far